material.py

```diff
@@ -135,8 +147,26 @@ def __derive_thermal__(self):
 
         # Slice scattering matrix based on g_thermal
         thermal = self.xsec['sig_s'][i:, i:]
-        self.derived.update({'ksi_ua': np.linalg.eigvals(thermal)})
+        th_d_i  = np.tril(thermal)
+        th_u    = np.triu(thermal, 1)
+        
+        total   = np.diag(self.xsec['sig_t'][i:])
+        try:
+            M = np.matmul(np.linalg.inv(total - th_d_i), th_u)
+            w,v = np.linalg.eig(M)
+            ksi_ua = v[:, np.argmax(np.absolute(w))]
```

Comment:
Would you point me to the normalization in case you implemented it but i skipped. @jsrehak

Reply:
Of course! `linalg.eig` returns normalized eigenvectors.
> The normalized (unit “length”) eigenvectors, such that the column `v[:,i]` is the eigenvector corresponding to the eigenvalue `w[i]`

[link](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.linalg.eig.html)